CMakeLists.txt

```diff
@@ -13,12 +13,18 @@ if(POLICY CMP0015)
 	cmake_policy(SET CMP0015 OLD)
 endif()
 
+# allow relative paths in LINK_DIRECTORIES
+if(POLICY CMP0081)
+	cmake_policy(SET CMP0081 OLD)
+endif()
 
 ###
 ### Project settings
 ###
 project(PINPROC)
 
+set (CMAKE_CXX_STANDARD 11)
+
```

Comment:
This addition was for updated `yaml-cpp`.  I may revert the change after removing the `yaml-cpp` dependency.